rizwan/infra/requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e .
+aws-cdk.core==1.122.0
+aws-cdk.aws_lambda==1.122.0
+aws-cdk.aws_events==1.122.0
+aws-cdk.aws_events_targets==1.122.0
```

Comment:
I think you should mention the datetime, and urllib3 library over here along with their versions that you are using.